index.html

```diff
@@ -2,9 +2,130 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="./styles/styles.css">
 </head>
 <body>
-
+    <header>
+        <div class="outer-container">
+            <nav class="inner-container-large-nav">
+                <ul>
+                    <li>Case Studies</li>
+                    <li>Services</li>
+                    <li>Contact Us</li>
+                </ul>
+            </nav>
+        </div>
+        <div class="outer-container">
+            <div class="inner-container-small introblock">
+                <h1>Your website is an investment in the  success of your business</h1>
+                <p>We are here to help you succeed</p>
+                <button>LET'S WORK TOGETHER</button>
+                <img src="assets/squiggle_line_stripe.svg" alt="------" />
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="outer-container">
+            <div class="inner-container-small">
+                <div class="intro-inner-container">
+                    <p class="intro-text">Rekupe is a digital agency in Los Angeles.</p>
+                    <p class="intro-text">We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</p>
+                    <p class="intro-text">We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</p>
+                    <p id="question-intro">Have a question? <a href="#">Click here</a> to contact us!</p>
+                </div>
+            </div>
+        </section>
+        <section class="outer-container casestudies">
+            <div class="inner-container-large">
+                <article class="case-study">
+                    <div class="case-study-text">
+                        <h2>Rolling Stone</h2>
+                        <img src="assets/squiggle_line.svg" alt="------" />
+                        <p>You know Rolling Stone. It's the definitive voice in music, politics and culture. It's a global icon.</p>
+                        <a href="#">VIEW PROJECT</a>
+                    </div>
+                    <div class="case-study-image">
+                        <img src="assets/work-image-01.jpeg" alt="Rolling Stone Image" />
+                    </div>
+                </article>
+                <article id="reverse" class="case-study">
+                    <div class="case-study-text">
+                        <h2>Variety</h2>
+                        <img src="assets/squiggle_line.svg" alt="------" />
+                        <p>Variety is an industry leader for breaking entertainment news, reviews and box office results with archive of content dating back to 1905.</p>
+                        <a href="#">VIEW PROJECT</a>
+                    </div>
+                    <div class="case-study-image">
+                        <img src="assets/work-image-01.jpeg" alt="Rolling Stone Image" />
```

Comment:
Probeer in de alt uitgebreider te omschrijven wat er wordt getoond in de afbeeldingen. De alt is er voor mensen met beperkt zicht zodat voor hen ook duidelijk wordt wat er in de afbeelding wordt getoond.